tests/integ/test_model_monitor.py

```diff
@@ -1852,3 +1852,197 @@ def _verify_default_monitoring_schedule_with_batch_transform(
         )
     else:
         assert network_config is None
+
+
+@pytest.mark.release
```

Comment:
Since this integ is added with release tag meaning it will be running as part of all regional integs (not just PDX), do we need to skip for nonsupported regions?
https://github.com/aws/sagemaker-python-sdk/blob/master/tests/integ/test_model_monitor.py#L282